packages/shared/src/components/fields/SearchField.tsx

```diff
@@ -136,10 +147,10 @@ export const SearchField = forwardRef(function SearchField(
             aria-hidden
             className="icon mr-2 text-2xl"
             role="presentation"
-            secondary={focused}
+            secondary={focused || isFocusVersion}
             style={{
               color:
-                focused || hasInput
+                focused || hasInput || isFocusVersion
```

Comment:
I agree, there should be other ways to make search more prominent without forcing focus state.